docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md

```diff
@@ -12,7 +12,7 @@ ms.assetid: f7f62f53-e026-4c40-b221-3a26adb0c2c5
 The following C# code example illustrates how to use platform invoke services to play a wave sound file on the Windows operating system.  
   
 ## Example  
- This example code uses `DllImport` to import `winmm.dll`'s `PlaySound` method entry point as `Form1 PlaySound()`. The example has a simple Windows Form with a button. Clicking the button opens a standard windows <xref:System.Windows.Forms.OpenFileDialog> dialog box so that you can open a file to play. When a wave file is selected, it is played by using the `PlaySound()` method of the winmm.DLL assembly method. For more information about winmm.dll's `PlaySound` method, see [Using the PlaySound function with Waveform-Audio Files](https://msdn.microsoft.com/library/aa910379.aspx). Browse and select a file that has a .wav extension, and then click **Open** to play the wave file by using platform invoke. A text box shows the full path of the file selected.  
+ This example code uses `DllImport` to import `winmm.dll`'s `PlaySound` method entry point as `Form1 PlaySound()`. The example has a simple Windows Form with a button. Clicking the button opens a standard windows <xref:System.Windows.Forms.OpenFileDialog> dialog box so that you can open a file to play. When a wave file is selected, it is played by using the `PlaySound()` method of the winmm.DLL assembly method. For more information about winmm.dll's `PlaySound` method, see [Using the PlaySound function with Waveform-Audio Files](https://docs.microsoft.com/windows/desktop/multimedia/using-playsound-to-play-waveform-audio-files). Browse and select a file that has a .wav extension, and then click **Open** to play the wave file by using platform invoke. A text box shows the full path of the file selected.  
```

Comment:
Sorry, @nemrism. My comment was evidently lost. This is unrelated to your PR, but it's a very noticeable error in the original text. winmm.dll is neither an assembly nor a method. Could you change "of the winmm.DLL assembly method" to read "in the winmm.dll assembly"?

Reply:
As `winmm.dll` isn't an assembly nor a method,  should we rather change it to read "of the `winmm.dll` library"?

Reply:
Either "in" of "of" is fine.